### PR TITLE
Translog: make sure stats's op count and size are in sync

### PR DESCRIPTION
**Pull request by bleskes**
During translog flush there is a very small window where translog stats reports can be inconsistent.

Example:

```
"translog" : {
   "operations" : 37415948,
   "size_in_bytes" : 0
},
```

### CODE REVIEW

**Comment by dakrone** [comment]
LGTM

File: src/main/java/org/elasticsearch/index/translog/fs/FsTranslog.java [hunk 1]
```diff
@@ -466,6 +466,11 @@ public void syncOnEachOperation(boolean syncOnEachOperation) {
 
     @Override
     public TranslogStats stats() {
-        return new TranslogStats(estimatedNumberOfOperations(), translogSizeInBytes());
+        FsTranslogFile current1 = this.current;
```
Comment (s1monw) [inline src/main/java/org/elasticsearch/index/translog/fs/FsTranslog.java]: can we name this just `current` I think this 1 is not necessary!

**Comment by s1monw** [comment]
left one comment

File: src/main/java/org/elasticsearch/index/translog/fs/FsTranslog.java (hunk 1 above)
Comment (bleskes) [inline src/main/java/org/elasticsearch/index/translog/fs/FsTranslog.java]: fair enough. I used the same pattern used in the file in other places.

**Pull request closed without merging**